### PR TITLE
Mock

### PR DESCRIPTION
**Pull request by Luceium**
Mocks many server actions and unwraps a redundant server action that wraps a route.
Also some small changes to combo box.

FYI, while server actions are cleaner to use, wrapping a route isn't beneficial. Additionally, route handlers are preferred for simple fetch and `GET` requests. All server actions are `POST` requests.

### CODE REVIEW

**Comment by Luceium** [comment]
@ryanhtang this PR will allow us to continue working on the payment integration with points.

**Comment by ray-sjsu** [comment]
https://github.com/KingSand08/LikeHome/pull/114

Should we merge this branch first? It does fix a lot of issues. However, different schema and had to change the region API output to be consistent with prisma schema.
Data is already prepopulated with cached regions.

File: prisma/schema.prisma [hunk 1]
```diff
@@ -53,3 +53,74 @@ type TransactionInfo {
...
+
+// Caching
+model CachedHotel {
+  id              String                 @id @default(auto()) @map("_id") @db.ObjectId
+  hotel_id        String                 @unique
+  tagline         String
+  location        Location
+  images          Image[]
+  hotelRoomOffers CachedHotelRoomOffer[]
+}
```
Comment (ray-sjsu) [inline prisma/schema.prisma:95]: This is missing `name`? The name of the hotel.

Comment (ray-sjsu) [inline prisma/schema.prisma:95]:
```suggestion
model CachedHotel {
  id              String                 @id @default(auto()) @map("_id") @db.ObjectId
  hotel_id        String                 @unique
  name        String
  tagline         String
  location        Location
  images          Image[]
  hotelRoomOffers CachedHotelRoomOffer[]
}
```

**Review by ray-sjsu — DISMISSED** [review]
Revise `prisma.schema`. `CachedHotels` is missing `name`.

**Comment by Luceium** [comment]
The cached hotels portion of the schema was not modified in this PR. I believe this was introduced in #111. As such if everything works as intended, I would not want to hold back this PR. @ray-sjsu since you introduced those changes I think you could update the schema and update all necessary references the fastest. Could you do this in a separate PR?

**Review by ray-sjsu — APPROVED** [review]

**Pull request merged**